src/main/java/seedu/address/logic/commands/ClearListingCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Listings;
+import seedu.address.model.Model;
+
+/**
+ * Clears all listings in the system.
+ */
+public class ClearListingCommand extends Command {
+
+    public static final String COMMAND_WORD = "clearListings";
+    public static final String MESSAGE_SUCCESS = "All listings have been cleared!";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.setListings(new Listings());
```

Comment:
Noticed that you used the same design principles as the previous ClearCommand. Good work!